src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/Tensor.cs

```diff
@@ -99,7 +99,7 @@ static Tensor<T> ITensor<Tensor<T>, T>.Create(scoped ReadOnlySpan<nint> lengths,
         {
             nint linearLength = TensorSpanHelpers.CalculateTotalLength(lengths);
             T[] values = pinned ? GC.AllocateArray<T>((int)linearLength, pinned) : (new T[linearLength]);
-            return new Tensor<T>(values, lengths.ToArray(), pinned);
+            return new Tensor<T>(values, lengths.ToArray(), 0, pinned);
```

Comment:
>0 [](http://example.com/codeflow?start=60&length=1)

nit: could you please refix this parameter with `memoryOffset: ` in all usages for the sake of readability?

Reply:
Fixed.